src/istgtcontrol.c

```diff
@@ -600,96 +605,95 @@ exec_refresh(UCTL_Ptr uctl)
-        rc = uctl_writeline(uctl);
-        if (rc != UCTL_CMD_OK) {
-                return rc;
-        }
-
-        /* receive result */
-        rc = uctl_readline(uctl);
-        if (rc != UCTL_CMD_OK) {
-                return rc;
-        }
-        arg = trim_string(uctl->recvbuf);
-        result = strsepq(&arg, delim);
-        strupr(result);
-        if (strcmp(result, "OK") != 0) {
-                if (is_err_req_auth(uctl, arg))
-                        return UCTL_CMD_REQAUTH;
-                fprintf(stderr, "ERROR %s\n", arg);
-                return UCTL_CMD_ERR;
-        }
-        return UCTL_CMD_OK;
+	const char *delim = ARGS_DELIM;
+	char *arg;
+	char *result;
+	int rc;
+
+	/* send command */
+	if (uctl->iqn == NULL || uctl->lun < 0) {
+			return (UCTL_CMD_ERR);
+		}
+		uctl_snprintf(uctl, "SYNC \"%s\" %d\n", uctl->iqn, uctl->lun);
+		rc = uctl_writeline(uctl);
+		if (rc != UCTL_CMD_OK) {
+			return (rc);
+		}
+
+		/* receive result */
+		rc = uctl_readline(uctl);
+		if (rc != UCTL_CMD_OK) {
+			return (rc);
+		}
+		arg = trim_string(uctl->recvbuf);
+		result = strsepq(&arg, delim);
+		strupr(result);
+		if (strcmp(result, "OK") != 0) {
+			if (is_err_req_auth(uctl, arg))
+			return (UCTL_CMD_REQAUTH);
+			fprintf(stderr, "ERROR %s\n", arg);
+			return (UCTL_CMD_ERR);
+		}
+		return (UCTL_CMD_OK);
 }
 
 static int
 exec_persist(UCTL_Ptr uctl)
 {
-        const char *delim = ARGS_DELIM;
-        char *arg;
-        char *result;
-        int rc;
-
-        /* send command */
-        if (uctl->iqn == NULL || uctl->lun < 0) {
-                return UCTL_CMD_ERR;
-        }
-        uctl_snprintf(uctl, "PERSIST \"%s\" %d %d\n"
-        ,uctl->iqn, uctl->lun, uctl->persistopt);
-        rc = uctl_writeline(uctl);
-        if (rc != UCTL_CMD_OK) {
-                return rc;
-        }
-
-        /* receive result */
-        rc = uctl_readline(uctl);
-        if (rc != UCTL_CMD_OK) {
-                return rc;
-        }
-        arg = trim_string(uctl->recvbuf);
-        result = strsepq(&arg, delim);
-        strupr(result);
-        if (strcmp(result, "OK") != 0) {
-                if (is_err_req_auth(uctl, arg))
-                        return UCTL_CMD_REQAUTH;
-                fprintf(stderr, "ERROR %s\n", arg);
-                return UCTL_CMD_ERR;
-        }
-        return UCTL_CMD_OK;
+	const char *delim = ARGS_DELIM;
+	char *arg;
+	char *result;
+	int rc;
+
+	/* send command */
+	if (uctl->iqn == NULL || uctl->lun < 0) {
+		return (UCTL_CMD_ERR);
+	}
+	uctl_snprintf(uctl, "PERSIST \"%s\" %d %d\n", uctl->iqn,
+	uctl->lun, uctl->persistopt);
+	rc = uctl_writeline(uctl);
+	if (rc != UCTL_CMD_OK) {
+		return (rc);
+	}
+
+	/* receive result */
+	rc = uctl_readline(uctl);
+	if (rc != UCTL_CMD_OK) {
+		return (rc);
+	}
+	arg = trim_string(uctl->recvbuf);
+	result = strsepq(&arg, delim);
+	strupr(result);
+	if (strcmp(result, "OK") != 0) {
+		if (is_err_req_auth(uctl, arg))
+		return (UCTL_CMD_REQAUTH);
```

Comment:
an extra tab is required here. Same for next 'if' also

Reply:
Done